### DISCUSSION TITLE
Outline container crushes in k8s with UPDATE_KEY error

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#3338] [Self Hosting]
Hello!
Can someone help me with outline crush, please.
I'm trying to deploy outline with minio in k8s cluster with external (outside of k8s) postgres and redis.
When i tried to deploy outline with command "yarn sequelize:migrate --env production-ssl-disabled && yarn start" i've got an error:

yarn run v1.22.17
$ sequelize db:migrate --env production-ssl-disabled

Sequelize CLI [Node: 16.14.0, CLI: 6.3.0, ORM: 6.9.0]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".
No migrations were executed, database schema was already up to date.
Done in 0.84s.
yarn run v1.22.17
$ node ./build/server/index.js
{"label":"lifecycle","level":"info","message":"Note: Restricting process count to 1 due to use of collaborative service"}
{"label":"lifecycle","level":"info","message":"\nIs your team enjoying Outline? Consider supporting future development by sponsoring the project:\n\nhttps://github.com/sponsors/outline\n"}
/opt/outline/node_modules/redis-parser/lib/parser.js:179
    return new ReplyError(string)
           ^

ReplyError: ERR unknown command `del`, with args beginning with: `UPDATES_KEY`,
    at parseError (/opt/outline/node_modules/redis-parser/lib/parser.js:179:12)
    at parseType (/opt/outline/node_modules/redis-parser/lib/parser.js:302:14) {
  command: { name: 'del', args: [ 'UPDATES_KEY' ] }
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Can you please tell me what is the possible cause of this error?

**Comment by tommoor** [comment]
Something in your Redis setup is awry – it's complaining that the standard command `del` is missing – did you rename it?

**Reply by ghost** [reply]
Hello. 
Thank you for your reply. There was a problem in redis indeed.
